app/src/main/java/app/revanced/manager/ui/screen/settings/AdvancedSettingsScreen.kt

```diff
@@ -82,15 +90,6 @@ fun AdvancedSettingsScreen(
                 }
             )
 
-            val exportDebugLogsLauncher =
-                rememberLauncherForActivityResult(ActivityResultContracts.CreateDocument("text/plain")) {
-                    it?.let(vm::exportDebugLogs)
-                }
-            SettingsListItem(
-                headlineContent = stringResource(R.string.debug_logs_export),
-                modifier = Modifier.clickable { exportDebugLogsLauncher.launch(vm.debugLogFileName) }
-            )
```

Comment:
Why was this removed?

Reply:
It's not removed, it's relocated

Reply:
Oh, it looked like it was removed in the diff but now I see it is there.